apps/ledger-live-mobile/src/screens/OperationDetails/Content.tsx

```diff
@@ -128,27 +139,46 @@ export default function Content({
   const uniqueRecipients = uniq<typeof operation.recipients[0]>(
     operation.recipients,
   );
-  const { extra, type } = operation;
-  const { hasFailed } = operation;
+  const { extra, type, hasFailed } = operation;
   const subOperations = operation.subOperations || [];
   const internalOperations = operation.internalOperations || [];
   const shouldDisplayTo = uniqueRecipients.length > 0 && !!uniqueRecipients[0];
+
+  const onEditTxPress = () => {
+    navigation.navigate(NavigatorName.EthereumEditTransaction, {
+      screen: ScreenName.EditEthereumTransactionMethodSelection,
+      params: { operation, account, parentAccount },
+    });
+  };
+
   const isConfirmed = isConfirmedOperation(
     operation,
     mainAccount,
     currencySettings.confirmationsNb,
   );
+
+  const isOperationStuck =
+    operation.date.getTime() <=
+    new Date().getTime() - getEnv("ETHEREUM_STUCK_TRANSACTION_TIMEOUT");
+
+  const isEditable =
+    mainAccount.currency.family === "ethereum" &&
+    !isConfirmed &&
+    operation.type === "OUT";
```

Comment:
Here as well, you might want to use the common method `isEditableOperation` from Huiqi